src/App.tsx

```diff
@@ -18,6 +19,15 @@ const App: FC = () => (
       <Route path={LOGIN_PAGE} exact>
         <Login />
       </Route>
+      <Route path={PROFILE} exact>
+        <Profile
+          userName="jeff"
+          userLastname="jefferson"
+          email="yo@gmail.com"
+          teamName="genericTeam"
+          developerRole="junior"
+        />
```

Comment:
I do not recommend passing this data as props, because I'm pretty sure Profile component will fetch data inside and set that data to state(pretty sure 100% sure 😆 ). So, for now, you can create simply mock user const in Profile component:

```ts
const Profile: FC = () => {
  const classes = useStyles()

  const user: User = { 
    userName: "jeff",
    userLastname: "jefferson",
    email: "yo@gmail.com",
    teamName: "genericTeam",
    developerRole: "junior",
  }

...
```

This way you do not leave yourself chores to remove unnecessary logic when API calls are implemented 😉

Reply:
Also, `User` type will most likely be used in several places throughout the whole app. So IMHO you should first of all create `User` type(also use enum for developer role):
```ts
enum UserRole = {
  junior = 'junior',
  mid = 'mid',
  senior = 'senior',
  teamLead = 'teamLead',
  god = 'god',
}

type User = {
  name: string
  lastname: string
  email: string
  teamName: string
  developerRole: UserRole
}
```

And then place it in some general directory - for example create `types/models/user.ts`. This way you can easily find types used throughout the whole app 👍.